### DISCUSSION TITLE
Control D and the  WinSecureDNSMgr

### DISCUSSION THREAD

**Discussion by ghost** [HotCakeX/Harden-Windows-Security#374] [Q&A]
Hello! 

How would I go about putting in the correct information to use the Control D DOH OR DOT DNS information into the module?

Which one should I use? I believe Control D is like NextDNS

Set-CustomWinSecureDNS -DoHTemplate https://example.com/
Set-CDOH -DoHTemplate https://example.com/
Set-CDOH -DoHTemplate https://example.com -IPV4s 1.2.3.4 -IPV6s 2001:db8::8a2e:370:7334

Sorry kind of new to all this!

Thank you!

**Comment by HotCakeX** [comment]
Hi,
sorry for the late response,
`Set-CDOH` and `Set-CustomWinSecureDNS` are the same commands, the first one is the shortened version (aka alias) so you don't have to type or remember a long command.

so between the last 2 options, it's up to you, if you define the IP address then it will use them, if you don't then the IP addresses of the DoH server will be detected automatically.

More examples here: https://github.com/HotCakeX/WinSecureDNSMgr?tab=readme-ov-file#custom-doh-examples

**Reply by ghost** [reply]
Thank you so much!

**Reply by HotCakeX** [reply]
You're welcome :)
